src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -806,7 +806,7 @@ private function deleteDeployment(callable $getProjectDB, callable $getFunctions
          * Delete builds
          */
         Console::info("Deleting builds for deployment " . $deploymentId);
-        $storageBuilds = $$getBuildsDevice($projectId);
+        $storageBuilds = $getBuildsDevice($projectId);
```

Comment:
uff, did that even work in the first place?

Reply:
Surprisingly this didn't crash the worker 😅 and only failed when we deleted a deployment